service/operator/service.go

```diff
@@ -0,0 +1,84 @@
+package operator
```

Comment:
I think this is not necessary anymore. There are probably still operators who have this dedicated service but this is rather legacy. Check the kvm-operator for instance. It does not have this anymore.